CMakeLists.txt

```diff
@@ -296,13 +296,9 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(i.86|AMD64|x86_64|amd64)")
   endif()
 endif()
 
-if(CMAKE_MAJOR_VERSION GREATER 2)
-  include(ProcessorCount)
-  ProcessorCount(CORECOUNT)
-  if(CORECOUNT LESS 1)
-    set(CORECOUNT 1)
-  endif()
-else()
+include(ProcessorCount)
+ProcessorCount(CORECOUNT)
+if(CORECOUNT LESS 1)
```

Comment:
What is this used for? I think we should not rely on it in any code since the same PoCL binary can be invoked from different machines with different CPU core counts.

Reply:
this is only used in CMake, for running tests in parallel (`make check` and `make check_tier`)